css/main.css

```diff
@@ -0,0 +1,303 @@
+body {
+  font-family: Raleway;
+  margin: 0;
+  background-color: rgb(241, 241, 241);
+}
+
+a, a:visited, a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+/* h1 not bold standard */
+h1, h3 {
+  font-weight: normal;
+}
+
+h2 {
+  text-transform: uppercase;
+}
+
+h2 span {
+  font-weight: 100;
+}
+
+h3 {
+  font-size: 150%;
+}
+
+.up-button {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  width: 30px;
+  height: 30px;
+  background-color: white;
+  border: 1px solid black;
+  text-align: center;
+  line-height: 30px;
+}
+
+/* ----------------------------- */
+/* header menu */
+/* ----------------------------- */
+.wrapper-header {
+  border-bottom: 1px solid rgba(0,0,0,0.8);
+  box-shadow: 0px 4px 16px 0px rgba(0,0,0,0.35); 
+  padding-top: 10px;
+  background-color: white; 
+}
+
+header {
+  width: 70%;
+  background-color: white;
+  margin: 0 auto 10px;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between; /* horizontal spread*/
+}
+
+nav li {
+  display: inline-block;
+  margin: 0 10px;
+}
+
+.logo {
+  width: 50px;
+  height: 50px;
+  border-radius: 25px; 
+  float: left;
+  margin: 0 10px;
+}
+
+.logo-text { 
+  width: 250px;
+}
+
+.logo-text h1, .logo-text p {
+  margin: 0px 10px;
+}
+
+.hire-button a {
+  border: 2px solid black;
+  margin: 0;
+  padding: 12px;
+  vertical-align: center;
+  font-weight: bold;
+}
```

Comment:
Your hire button should be direct descendant of your nav.  That way it can also benefit from the flex space-around properties :)